src/test/java/addtoPlaylist.java

```diff
@@ -0,0 +1,50 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.openqa.selenium.interactions.Actions;
+
+import java.security.Key;
+import java.time.Duration;
+
+public class addtoPlaylist extends BaseTest {
```

Comment:
**According to the Java style guide:** 

1. Class names should be nouns, in mixed cases with the first letter of each internal word capitalized. Interfaces names should also be capitalized just like class names.
2. Use whole words and must avoid acronyms and abbreviations.

For example` public class PlaylistTest`

Reply:
Ok, Thank you

Reply:
> Ok, Thank you

I'm glad you found it helpful